config/master/aws-k8s-cni-cn.yaml

```diff
@@ -168,8 +168,6 @@ spec:
               value: "true"
             - name: AWS_VPC_ENI_MTU
               value: "9001"
-            - name: AWS_VPC_K8S_CNI_CONFIGURE_RPFILTER
```

Comment:
Let's just make sure this captured in the release notes and any other EKS documentations.

Reply:
Yep, this is captured in release notes, in changelog, and in README. If customers set it in v1.12.1+, it is harmless anyway